src/matchingTag.ts

```diff
@@ -0,0 +1,147 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@NikolasKomonen it seems that you are using an old code (which I think that there are some bugs). IMHO you should use https://github.com/microsoft/vscode/blob/master/extensions/html-language-features/client/src/mirrorCursor.ts